src/boot/efi/util.c

```diff
@@ -85,6 +85,19 @@ EFI_STATUS efivar_set_uint64_le(const EFI_GUID *vendor, const char16_t *name, ui
         return efivar_set_raw(vendor, name, buf, sizeof(buf), flags);
 }
 
+EFI_STATUS efivar_unset(const EFI_GUID *vendor, const char16_t *name, uint32_t flags) {
+        EFI_STATUS err;
+
+        assert(vendor);
+        assert(name);
+
+        err = efivar_get_raw(vendor, name, NULL, NULL);
+        if (err == EFI_SUCCESS)
+                err = efivar_set_raw(vendor, name, NULL, 0, flags);
```

Comment:
(We prefer early returns.)
```suggestion
                return efivar_set_raw(vendor, name, NULL, 0, flags);
```